### DISCUSSION TITLE
Reliability and troubleshooting

### DISCUSSION THREAD

**Discussion by agrieco** [gtxaspec/wz_mini_hacks#742] [Q&A]
Hi,

New to using the project and have it installed on 3 v3s.  I downgraded my cams and it is up and running and I get RTSP feeds (go2rtc) and SSH into the cameras and the feeds go into Frigate great.  Life is good.  However, after some period of time (less then 2 days typically), the cameras go offline, are no longer reachable via SSH, doesnt answer a ping, webserver isn't reachable and RTSP stops.  If i pull the power and let it reboot- comes back up.

My question is- can someone walk me through diagnostic steps to go to from here to figure out what is going on?  Are there log files on the device that would help me sus this out?  What should I look for?

Background on my setup- All using wifi.  These cameras were deployed without wz_mini_hacks in exactly the same physical location as before for at least a year with no issues on a pure, clean Wyze install.

**Comment by rgviewer** [comment]
I am wondering the same, except I have 4 wyze v2 cams with frigate. I have a working USB TPU. At the moment I am using docker wyze bridge to send my rtsp to Firgate,  But I get a lot of errors in frigate logs about "Unable to read frames from ffmpeg process" and "rtsp://192.168.4.240:8554/porch: Invalid data found when processing input". Not sure why this is happening, but looking for ways to midigate these errors. Thinking of trying wz_mini_hacks to see if I get better results. 
Do you self host as well?
